charts/acp-cd/README.md

```diff
@@ -0,0 +1,70 @@
+# Cloudentity Authorization CD
+
+[Cloudentity](https://cloudentity.com/) Authorization Control Plane (ACP) is a cutting edge platform for the API access control. ACP consolidates capabilities of a modern OAuth/OIDC server with an advanced authorization, consent management, and developer enablement.
+
```

Comment:
Let's add more information on what this chart does:

`acp-cd` provides a mechanism allowing you to declaratively manage the configuration of your ACP data.
It allows to set up policies, APIs, clients, services, and all other features of ACP by defining those in a file.
It can be used to set up a GitOps-like pipeline, or ad-hoc import, easing the process.

Reply:
fixed